File: L10 - Blog/VUE/blog/src/store/entities/blog.ts
import Entity from './entity';

export default class Blog extends Entity<string> {
    public slug!: string;
    public title!: string;
    public summary!: string;
    public content!: string;
    public htmlContent!: string;
    public isShow!: boolean;
    public creationTime!: Date;
}

File: L10 - Blog/BlogApi/src/main/java/com/sea/blog/model/Blog.java
package com.sea.blog.model;

import java.util.Date;
import java.util.List;

import lombok.Data;

@Data
public class Blog {
    private String Id;
    private String CreatorUserId;
    private Date CreationTime;
    private String LastModifierUserId;
    private Date LastModificationTime;
    private String DeleterUserId;
    private Date DeletionTime;
    private Boolean IsDeleted;
    private String Slug;
    private String Title;
    private String Summary;
    private String Content;
    private Boolean IsShow;
    private List<BlogCategory> blogCategories;
}